python/pyspark/rdd.py

```diff
@@ -152,6 +152,8 @@ class PythonEvalType:
     SQL_MAP_ARROW_ITER_UDF: "ArrowMapIterUDFType" = 207
     SQL_GROUPED_MAP_PANDAS_UDF_WITH_STATE: "PandasGroupedMapUDFWithStateType" = 208
 
+    SQL_TABLE_UDF = 300
```

Comment:
Mind adding a literal at `python/pyspark/sql/pandas/_typing/__init__.pyi` and use its type definition?